en/docs/administer/changing-to-ibm-db2.md

```diff
@@ -1,84 +1,155 @@
 # Changing to IBM DB2
 
-The following sections describe how to replace the default H2 database
-with IBM DB2:
-
-!!! tip "Before you begin"
-    You need to set up IBM DB2 before following the steps to configure your
-    product with it. For more information, see [Setting up IBM
-    DB2](../../administer/setting-up-ibm-db2).
+By default, WSO2 Identity Server uses the embedded H2 database as the database
+for storing user management and registry data. Given below are the steps
+you need to follow in order to use DB2 for this purpose.
     
 
-### Setting up datasource configurations
-
-A datasource is used to establish the connection to a database. By
-default, `         WSO2_CARBON_DB        ` datasource is used to connect
-to the default H2 database, which stores registry and user management
-data. After setting up the IBM DB2 database to replace the default H2
-database, configure a new datasource to point it to the new database as explained below.
-
-#### Changing the default datasource
+## Setting up datasource configurations
+
+A datasource is used to establish the connection to a database. By
+default, `WSO2_IDENTITY_DB` and `WSO2_SHARED_DB` datasources are used to connect
+to the default  H2 database. 
+
+- `WSO2_SHARED_DB` - The database which stores registry and user management
+                     data.
+- `WSO2_IDENTITY_DB` - The database specific for the identity server which stores
+                       identity related data
+                       
+After setting up DB2 database. You can point the `WSO2_IDENTITY_DB` or 
+`WSO2_SHARED_DB` or both to that DB2 database by following below instructions.
+
+### Changing the default datasource
+
+1.  **Minimum Configurations for changing default datasource to DB2.**
+ 
+ Configurations can be done by editing the default configurations in `<IS-HOME>/repository/conf/deployment.toml`. 
+ Following are the basic configurations and their descriptions. 
+      <table>
+      <thead>
+      <tr class="header">
+      <th>Element</th>
+      <th>Description</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr class="even">
+      <td><strong>username</strong> and <strong>password</strong></td>
+      <td>The name and password of the database user.</td>
+      </tr>
+      <tr class="even">
+      <td><strong>type</strong></td>
+      <td>The type of the database.</td>
+      </tr>
+      <tr class="even">
+      <td><strong>hostname</strong></td>
+      <td>The hostname of the host where database is hosted.</td>
```

Comment:
Shall we change this to "The hostname of the hosted database"

Reply:
done